vignettes/hazardItems.Rmd

```diff
@@ -156,5 +160,39 @@ thumb.service(serviceEndpoint)
 ```
 
 
+#Refresh/delete of various caches
+
+There are a number of functions that are required in order to clear out cached contents when items change. 
+
+##Delete Entire Application Tile Cache
+
+The `deleteTileCache` function takes the cache URL set by `pkg.env$tile_cache` and drops the entire tile cache for the application. The user must be authenticated to perform this task. 
+
+To use:
+```{r,eval=FALSE}
+setBaseURL('dev') #setBaseURL("prod") in the event of production use.
+tileCacheUrl <- pkg.env$tile_cache
+deleteTileCache(username='bbadger') #replace the user making the request, if not authenticated, the checkAuth function will be called
+```
+
+##Delete Download Cache 
+
+The `deleteDownload' function accepts the itemID for an item and drops the cached download for the item. The user must be authenticated to perform this task.
 
+To use:
+```{r,eval=FALSE}
+setBaseURL('dev') #setBaseURL("prod") in the event of production use.
+authenticateUser("bbadger")
+deleteDownload("CCNKrhr") #itemID you wish to delete the download cache for
+```
+
+##Thumbnail Refresh
+
+The `refreshItemThumbnail` accepts an item ID and will verify that the item exists, before putting a newly created thumbnail in the database.
+
+To use:
+```{r,eval=FALSE}
+setBaseURL('dev') #setBaseURL("prod") in the event of production use.
+refreshItemThumbnail('CCGftiy') #or whichever itemID you wish to regenerate and put
+```
 
```

Comment:
Good. And then should we have a higher-level workflow for the various steps required to keep things current when someone changes an item or an aggregation? It includes a number of these (refreshThumb, killCache if needed, kill downloads, re-trigger downloads etc)